iroh-metrics/Cargo.toml

```diff
@@ -53,8 +52,7 @@ features = [
 optional = true
 
 [features]
-default = ["rpc-grpc"]
-rpc-grpc = ["tonic"]
+default = []
```

Comment:
What's the reason to leave these as empty btw?  Does it do something special?

Reply:
No, just made this empty instead of fully removing it. We got those all over the place. Want me to remove them all in this PR?

Reply:
Personally I would clean them up, but I don't mind.  I was mostly curious if I missed something about what their presence achieved.